sites/bizbash/browser/image-slide.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="carousel-item" :class="classNames">
+    <img class="d-block w-100" :src="image.src" :alt="image.alt" />
+    <div class="carousel-caption d-none d-md-block">
+      <h5 v-if="image.displayName">{{ image.displayName }}</h5>
+      <p v-if="image.caption">{{ image.caption }}</p>
+      <p v-if="image.credit">{{ image.credit }}</p>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.carousel-caption {
```

Comment:
Because we currently do not use styling in Vue components, I would prefer this to be moved to the theme and controlled there.